clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/description/servergroup/DeployKubernetesAtomicOperationDescription.groovy

```diff
@@ -96,6 +97,37 @@ class KubernetesContainerDescription {
 
 @AutoClone
 @Canonical
+class KubernetesDeployment {
+  boolean enabled
+  KubernetesStrategy deploymentStrategy
+  int minReadySeconds
```

Comment:
Is there a reason to alternate between the primitive and wrapper types here? If so, please leave a comment or two for future readers.